src/odemis/gui/util/img.py

```diff
@@ -720,10 +720,10 @@ def calculate_raw_ar(data, bg_data):
         # Simple version: remove the background value
         data0 = angleres.ARBackgroundSubtract(data)
     else:
-        data0 = img.Subtract(data, bg_data)  # metadata from data
+        # substract bg image, but don't clip (keep negative values for export)
+        data0 = (data.astype('float64') - bg_data.astype('float64'))  # metadata from data
```

Comment:
In general, it's a little safer to write the dtypes as `numpy.float64`, instead of a string. In case of a typo, it'll be reported earlier, and more clearly.

Also, you don't need the main parenthesis.